enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -108,14 +109,23 @@ int enclave_tls_echo(int fd, enclave_tls_log_level_t log_level,
 		printf("\n");
 
 		printf("Server:\n%s\n", buf + 2 * sizeof(sgx_measurement_t));
-	} else {
+	} else
+#endif
+	{
 		/* Server not running in SGX Enlcave will only send hello message to client */
 		printf("Server: %s\n", buf);
 	}
 
+	if (strcmp(msg, buf)) {
+		fprintf(stderr, "Invalid response retrieved from enclave-tls server\n");
+		goto err;
+	}
+
 	ret = enclave_tls_cleanup(handle);
-	if (ret != ENCLAVE_TLS_ERR_NONE)
+	if (ret != ENCLAVE_TLS_ERR_NONE) {
 		fprintf(stderr, "failed to cleanup %#x\n", ret);
+		goto err;
```

Comment:
Since `enclave_tls_cleanup` is failed, `goto err` to call `enclave_tls_cleanup` twice is Meaningless

Reply:
OK.